README.md

```diff
@@ -78,6 +78,11 @@ Some users of the technologies developed in Kratos are:
 <img align="left" src="https://github.com/KratosMultiphysics/Documentation/raw/master/Wiki_files/Logos/onera_logo.png" width="128">
 <p>ONERA, The French Aerospace Lab<br>Applied Aerodynamics Department</p>
 </span>
+<span>
+<img align="left" src="https://github.com/KratosMultiphysics/Documentation/raw/master/Wiki_files/Logos/Deltares_logo.png" width="128">
```

Comment:
Can I say the resolution is not very HD?

Reply:
Its is not, Its optimized for 128 pixels